content/02.main-text.md

```diff
@@ -47,23 +47,23 @@ Although they work well for editing text, they lack sufficient features for mana
 | Editing software | Any text editor | Web interface | Web interface | Web interface | Web interface | Any text editor |
 | Document format | Markdown | LaTeX | LaTeX | Proprietary | Proprietary | Markdown |
 | Templating | Yes | Yes | Yes | No | No | No |
-| Technial expertise required | Yes | Yes | Yes | No | No | Yes |
+| Technical expertise required | Yes | Yes | Yes | No | No | Yes |
 
 Table: Collaborative writing platforms.
 A summary of features that differentiate Manubot from existing collaborative writing platforms.
 We assessed features on June 15, 2018 using the free version of each platform.
 Some platforms offer additional features through a paid subscription or software.
 <sup>1</sup>Additional functionality, such as bibliography management, is available by editing the Word document stored in OneDrive with the paid Word desktop application.
 <sup>2</sup>Conversations about modifications take place on the document as comments, annotations, or unsaved chats.
-There is no separate integrated forum for discussing and editing revisions.
-<sup>3</sup>Simultaneous edits online can be attributed to a single author in a git commit.
+There is no integrated forum for discussing and editing revisions.
+<sup>3</sup>In some circumstances, a git commit containing different edits made at the same time by distinct authors may be attributed to a single author.
```

Comment:
I struggled with how to word this. But the essence is that authorship gets assigned to the person who makes the commits, not the person who makes the edits.